ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml

```diff
@@ -378,26 +378,30 @@ objects:
           oc --insecure-skip-tls-verify --request-timeout=5s get pods --all-namespaces --template '{{ range .items }}{{ $name := .metadata.name }}{{ $ns := .metadata.namespace }}{{ range .spec.containers }}-n {{ $ns }} {{ $name }} -c {{ .name }}{{ "\n" }}{{ end }}{{ range .spec.initContainers }}-n {{ $ns }} {{ $name }} -c {{ .name }}{{ "\n" }}{{ end }}{{ end }}' > /tmp/containers
           oc --insecure-skip-tls-verify --request-timeout=5s get pods -l openshift.io/component=api --all-namespaces --template '{{ range .items }}-n {{ .metadata.namespace }} {{ .metadata.name }}{{ "\n" }}{{ end }}' > /tmp/pods-api
 
+          for group in config.openshift.io operator.openshift.io; do
+              queue /tmp/artifacts/"$(echo ${group} | cut -f 1 -d .)"-resources.json oc --insecure-skip-tls-verify --request-timeout=5s get $(oc api-resources --api-group ${group} -o name | tr '\n' ',' | sed 's/.$//') -o json;
+          done
           queue /tmp/artifacts/apiservices.json oc --insecure-skip-tls-verify --request-timeout=5s get apiservices -o json
-          queue /tmp/artifacts/clusteroperators.json oc --insecure-skip-tls-verify --request-timeout=5s get clusteroperators -o json
-          queue /tmp/artifacts/clusterversion.json oc --insecure-skip-tls-verify --request-timeout=5s get clusterversion -o json
           queue /tmp/artifacts/configmaps.json oc --insecure-skip-tls-verify --request-timeout=5s get configmaps --all-namespaces -o json
+          queue /tmp/artifacts/credentialsrequests.json oc --insecure-skip-tls-verify --request-timeout=5s get credentialsrequests --all-namespaces -o json
```

Comment:
Does this contain secrets?

Reply:
Only secret refs from what I see: https://github.com/openshift/cloud-credential-operator/blob/master/pkg/apis/cloudcredential/v1/credentialsrequest_types.go